tests/pytest/test.py

```diff
@@ -835,7 +835,7 @@ def testLenCommands(env):
     r.expect('JSON.ARRLEN', 'test', '.arr[999]').raiseError().contains("does not exist")
 
     # test an infinite index
-    r.expect('JSON.ARRLEN', 'test', '.arr[-inf]').raiseError().contains("path error")
+    r.expect('JSON.ARRLEN', 'test', '.arr[-inf]').raiseError().contains("Error accured")
```

Comment:
Why not test also some of the error message details? (e.g., that it is a path related error)

Reply:
It is a detailed error that explain what it is expect to get. I believe this is good enough.